.pubnub.yml

```diff
@@ -1,9 +1,22 @@
 ---
 name: swift
 scm: github.com/pubnub/swift
-version: "1.1.0"
+version: "1.2.0"
 schema: 1
 changelog:
+  -
+    changes:
+      -
+        text: "Add support for Global HereNow"
+        type: bug
+      -
+        text: "Fixed HereNow stateless response decoding"
+        type: bug
+      -
+        text: "Fixed HereNow single channel response decoding"
+        type: bug
+    date: 2019-XX-XX
```

Comment:
Should probably update the date here.

Reply:
Oh are we doing this after release?

Reply:
Just whenever we know the date of release.  Typically whenever we make the release PR (if there is one), after the release, or before if the date is predictable.